### DISCUSSION TITLE
Closing tags for head, body and html are missing in the exported HtmlDark html

### DISCUSSION THREAD

**Discussion by yarhrn** [Tyrrrz/DiscordChatExporter#1182] [Q&A]
Hello, thanks for the excellent tool!

I have checked the HTML for validity and noticed that several closing tags are missing. Specifically, there appear to be no closing tags for head, body, and html. I am exporting using Docker with the latest Discord chat exporter.

Is this expected?

Thanks in advance.

**Comment by Tyrrrz** [comment]
No, it's not expected. At least if the export finished gracefully.
